Automattic/wp-calypso
# Email stats: change the routes, place site at the end

#### Proposed Changes

* This PR changes the routing for email stats:

| From | To |
|------|----|
| /stats/email/clicks/&lt;siteid&gt;/day | /stats/email/clicks/day/&lt;siteid&gt; | 
| /stats/email/clicks/&lt;siteid&gt;/hour | /stats/email/clicks/hour/&lt;siteid&gt; | 
| /stats/email/opens/&lt;siteid&gt;/day | /stats/email/opens/day/&lt;siteid&gt; | 
| /stats/email/opens/&lt;siteid&gt;/hour | /stats/email/opens/hour/&lt;siteid&gt; | 


#### Testing Instructions

* Apply this PR
* Start at Jetpack stats, and append `?flags=newsleter/stats` to the URL (click Stats in the sidebar)
* Make sure clicking a link in the Email Opens & Email Clicks panel takes you to a detail view of Opens & Clicks
* In the detail view make sure that switching to Hours for the chart works
* Also make sure that clicking the arrow keys works

#### Pre-merge Checklist





Related to #71523
